modules/javafx.graphics/src/main/native-glass/mac/GlassMacros.h

```diff
@@ -174,6 +174,10 @@ do {
     };                                                                             \
 } while (0)
 
+#define GLASS_CHECK_NONNULL_EXCEPTIONALLY_RETURN(ENV, EXPR)                        \
+    GLASS_CHECK_EXCEPTION(ENV);                                                    \
```

Comment:
Should this call `GLASS_CHECK_EXCEPTIONALLY_RETURN` instead? It doesn't matter for the current places where it is used, but it might be more consistent.

Reply:
Yes.